Felienne/hedy
# [FIX] Cleans up languages abbreviations to align YAMLs and Babel

**Description**
In this PR we clean-up the language abbreviations to align the keys used by the YAMLs and bij Babel. We correctly capitalize the second part of the Portuguese languages and change the Chinese translations from `zh` to `zh_Hans`, also fixing the bug where the Chinese Babel translations where not shown.

**Fixes**
This PR fixes #2332.

**How to test**
Verify that the Chinese UI translations are now working indeed. You can simply verify by noticing the translated menubar options. Otherwise this is a clean-up task, nothing broken: it works!
